client-src/elements/chromedash-header.js

```diff
@@ -245,18 +307,19 @@ export class ChromedashHeader extends LitElement {
     return html`
       ${this.user ? html`
         ${this.user.can_create_feature && !this.isCurrentPage('/guide/new') ? html`
-          <sl-button href="/guide/new" variant="primary" size="small">
+          <sl-button data-testid="create-feature-button"
+            href="/guide/new" variant="primary" size="small">
             Create feature
           </sl-button>
         `: nothing }
-        <div class="nav-dropdown-container">
+        <div class="nav-dropdown-container" data-testid="nav-container">
```

Comment:
The terms "nav" and "container" are pretty common.  I think of this as the "account indicator".

Reply:
Changed to account-indicator.

Reply:
Renamed.